### PR TITLE
App styling

### PR DESCRIPTION
**Pull request by dustin-oz**

### CODE REVIEW

File: app/javascript/components/components/Footer.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import { Container } from "react-bootstrap";
 // import { Container } from 'reactstrap'
```
Comment (archae0pteryx) [inline app/javascript/components/components/Footer.js:3]: delete unused code

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import React, { Component } from 'react'
-import LOGO from '../assets/joint4ces_logo.jpg'
-import { NavItem, Nav, NavbarBrand, Navbar } from 'reactstrap'
-import { NavLink } from 'react-router-dom'
+import React, { Component } from "react";
+// import LOGO from "../assets/joint4ces_logo.jpg";
```
Comment (archae0pteryx) [inline app/javascript/components/components/Header.js:2]: unused

File: app/javascript/components/pages/MyListing.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
 import React, { Component } from "react";
-import { Row, Col, Card, Button } from "reactstrap";
 import {NavLink} from 'react-router-dom'
+import { Container, Card, Row, Col, Button } from "react-bootstrap";
```
Comment (archae0pteryx) [inline app/javascript/components/pages/MyListing.js:3]: run your linter / formatter on this file. it's got a bunch of formatting issues.

File: app/javascript/components/components/Footer.js (hunk 1 above)
Comment (dustin-oz) [reply app/javascript/components/components/Footer.js]: Thank you. Resolved.

File: app/javascript/components/pages/About.js [hunk 4]
```diff
@@ -1,9 +1,121 @@
...
+                Service, and the Government of the Virgin Islands. Necocli is a
+                multi-passionate web developer who brings two decades of
+                military and government expertise to every project and her
+                unique voice to the tech industry.
+              </Card.Text>
+            </Card.Body>
+          </Card>
+        </div>
+
+        {/* <Row xs={1} md={2} className="g-4">
```
[the remark this answers is not in the document]
Comment (dustin-oz) [reply app/javascript/components/pages/About.js:109]: Resolved

File: app/javascript/components/pages/MyListing.js (hunk 3 above)
Comment (dustin-oz) [reply app/javascript/components/pages/MyListing.js:3]: Resolved.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**